tests/Transports.AspNetCore.Tests/NewtonsoftJsonTests.cs

```diff
@@ -41,64 +42,61 @@ public async Task Decodes_BigInteger()
             var request = @"{""variables"":{""a"":1234567890123456789012345678901234567890}}";
             var ret = await Deserialize(request);
             var bi = BigInteger.Parse("1234567890123456789012345678901234567890");
-            ret.Single.Inputs["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
+            ret.Single().Variables["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
         }
 
         [Fact]
-        public async Task Dates_Should_Parse_As_Text()
+        public async Task Dates_Should_Parse_As_Dates()
         {
             var ret = await Deserialize(@"{""variables"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Inputs["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
+            ret.Single().Variables["date"].ShouldBeOfType<DateTime>().ShouldBe(new DateTimeOffset(2015, 12, 22, 10, 10, 10, TimeSpan.FromHours(3)).LocalDateTime);
```

Comment:
The question about default behavior for dates left.

Reply:
What's the question?  Any change would need to be made in the main project.

Reply:
Yes, reviewed. Let's merge and update here.